auth_password_settings/data/auth_password_settings_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <!--SETTINGS-->
+        <record id="default_auth_password_min_character" model="ir.config_parameter">
+            <field name="key">auth_password_settings.auth_password_min_character</field>
+            <field name="value">4</field>
```

Comment:
It was 6 (like the default), but then Ttravis failed on other modules in this repos adding test accounts like demo/demo etc